kong/clustering/init.lua

```diff
@@ -216,6 +216,45 @@ function _M:request_version_negotiation()
   end
 end
 
+local function call_control_plane(premature, self)
+  if premature then
+    return
+  end
+
+  local wsjs_dp = require "kong.clustering.data_plane"
+  local wrpc_dp = require "kong.clustering.wrpc_data_plane"
+
+  ngx_log(ngx_DEBUG, _log_prefix, "will try wrpc first")
+
+  local implementation = wrpc_dp
+  local ws_conn, err = implementation.open_connection(self)
+  ngx_log(ngx_DEBUG, _log_prefix, "got conn: ", tostring(ws_conn), ", err: ", tostring(err))
+
+  -- TODO: find how to detect a 404 response
+  if not ws_conn and err == "404" then
+    ngx_log(ngx_DEBUG, _log_prefix, "retrying on wsjs")
```

Comment:
```suggestion
    ngx_log(ngx_DEBUG, _log_prefix, "wRPC not available, falling back to vanilla websocket")
```
wsjs is not a known term to most Kong engineers or our users.